### ISSUE TITLE
[BUG]if not configure -output-http-works, worker can not scale out automatically

### ISSUE THREAD

**Issue by liujin1993** [buger/goreplay#675]
If not configure -output-http-works and -output-http-works-min, both of them are 0, so worker can not scale out automatically, and it will always be 10,and send rps is very slow!

![image](https://user-images.githubusercontent.com/31949674/58542588-784f1300-8230-11e9-97db-ec05c18ab2ee.png)
i think it should be 
```
if o.config.workersMax != o.config.workersMin || o.config.workersMax == 0
```

**Comment by liujin1993** [comment]
@bruce34

**Comment by urbanishimwe** [comment]
this was fixed #833
